pepper-apis/parent-pom.xml

```diff
@@ -18,7 +18,7 @@
         <itext.version>7.1.4</itext.version>
         <itext.licensekey.version>3.0.3</itext.licensekey.version>
         <liquibase.version>4.8.0</liquibase.version>
-        <elasticsearch.version>6.8.15</elasticsearch.version>
+        <elasticsearch.version>7.17.10</elasticsearch.version>
```

Comment:
Are we upgrading in steps (i.e. to 7.x then 8.x)?

Reply:
yes Dennis. upgrading to 7.17 will bring it into supported version.